src/apps/eth/eth_verify.c

```diff
@@ -136,13 +137,19 @@ app_eth_sign_error_t app_eth_verify_standard_transaction(const ETHSignRequest* r
     if (params == NULL) {
         return APP_ETH_SIGN_ERR_INVALID_INPUT;
     }
-    if (request->value.size == 0) {
```

Comment:
unsure, but possibly value==0 and data.size==0 is invalid (must have at least one of both?)

Reply:
OK, added that case